### DISCUSSION TITLE
Will you make a new installer for MS Windows soon?

### DISCUSSION THREAD

**Discussion by beischer** [wxFormBuilder/wxFormBuilder#695] [General]
Seems like the latest installer is from 2018 but you continue to improve the code.

Magnus

**Comment by sodevel** [comment]
There won't be a new release until the important issues from this [milestone](https://github.com/wxFormBuilder/wxFormBuilder/milestone/10) are resolved. However, since there have been recent pushes, you can download a recent installer from the artifacts [here](https://github.com/wxFormBuilder/wxFormBuilder/actions/workflows/windows.yml).
